### PR TITLE
Enable users to provide feedback via chat

### PR DESCRIPTION
**Pull request by mushfick**

### CODE REVIEW

**Comment by tielur** [comment]
We have to be careful with stuff like this because it's not always an Alice issue. It could be the individual bots issue ( with for example hard coded routes), or separate handlers that are pulled out into a module, ect...

I do like the general idea of this though

**Comment by mushfick** [comment]
This handler is specifically for awesome_friendz slack, so we all have a convenient way of saving the great and wacky ideas we come up with for alice and any handlers.

**Comment by tielur** [comment]
Doh I should probably read what repo I'm on before commenting 😆 

👍 🚢

File: lib/alice/handlers/feedback.ex [hunk 1]
```diff
@@ -0,0 +1,29 @@
+defmodule Alice.Handlers.Feedback do
+  use Alice.Router
+  @moduledoc "Lets users provide feedback (by opening an issue in github) conveniently."
+
+  alias Alice.Conn
+
+  route ~r/\b(feedback|idea|bug|enhancement|suggestion) *for *alice *:?(.+)/i, :feedback
+
+  @doc "Opens a github issue with provided feedback"
```
Comment (adamzaninovich) [inline lib/alice/handlers/feedback.ex]: needs a usage example, see other handlers

Comment (mushfick) [reply lib/alice/handlers/feedback.ex]: added

**Comment by mushfick** [comment]
@adamzaninovich I've made the changes you requested. Anything else you'd like me to update on this?

**Pull request merged**